packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/GoogleMapCircleController.h

```diff
@@ -14,6 +14,10 @@
                               circleId:(NSString *)circleIdentifier
                                mapView:(GMSMapView *)mapView
                                options:(NSDictionary *)options;
+- (instancetype)initWithCircle:(GMSCircle *)circle
```

Comment:
Are these new initializers only here for tests? If so, they should be declared in `..._Test.h` files instead of here.

Reply:
Fixed